spec.emu

```diff
@@ -189,6 +190,7 @@ contributors: Ujjwal Sharma, Younies Mahmoud
     <emu-clause id="sec-getdurationunitoptions" type="abstract operation">
       <h1>
         GetDurationUnitOptions (
+          _durationFormat_: a DurationFormat Object,
```

Comment:
Why do we need to pass in a  DurationFormat Object here?  I see you only use _durationFormat_.[[TwoDigitHours]]  inside this AO. Why not just pass in that value only ?